### ISSUE TITLE
Please don't lock package requirements to specific versions

### ISSUE THREAD

**Issue by uSpike** [alexdelorenzo/aiopath#15]
`setup.py` reads `requirements.txt` to populate `install_requires`.  The issue is that typically `requirements.txt` requirements are locked to specific versions for reproducibility of developer environments.  This is not ideal for `install_requires` since these specific version dependencies will reduce the entire environment of someone who installs this package.

For example, once `anyio` 3.2 comes out, I have to wait for this project to update `requirement.txt` before I can use it in my project.

The solution is to update `setup.py` to be less specific about version numbers:

```
install_requires=["anyio>=3,<4", "aiofile>=3,<4"]
```

**Comment by alexdelorenzo** [comment]
Thanks for the tip, sorry for holding your project back!

This was addressed in https://github.com/alexdelorenzo/aiopath/commit/02ec0174644e9be9f29f5c3184300f5b0aab09a2 and https://github.com/alexdelorenzo/aiopath/commit/a988aa1917e268c6f810592bedef9b3e5c57ad33, and I'll comment again when the changes are pushed to PyPI.

**Comment by alexdelorenzo** [comment]
This issue was addressed in `aiopath==0.5.9` for Python 3.9 and below, and in `aiopath==0.6.9` for Python 3.10 and above.

You can upgrade `aiopath` like so:
```bash
python3 -m pip install --upgrade aiopath
```

**Comment by alexdelorenzo** [comment]
I'm going to close this issue. Feel free to comment again if there's still an issue.
